README.md

```diff
@@ -29,17 +29,70 @@ $ npm install
 
 ## Running the app
 
-```bash
-# development
-$ npm run start
-
-# watch mode
-$ npm run start:dev
-
-# production mode
-$ npm run start:prod
+- Using Docker
```

Comment:
What is the port that iam-org-creator is running on? Should it be passed as an env variable or via some other way?

Reply:
@KaamilW The port will be gotten from the env variables.